deployment/constructor_params/lynx/beta_program_initiator.yml

```diff
@@ -0,0 +1,20 @@
+deployment:
+  name: beta-program-initiator
+  chain_id: 80001
+
+artifacts:
+  dir: ./deployment/artifacts/
+  filename: beta_program_initiator_lynx.json
+
+constants:
+  # See deployment/artifacts/lynx.json
+  COORDINATOR_PROXY: "0x530608219a8A671FD183534b17E2a2CE09e782a4"
+
+  # lynx deployer account
+  EXECUTOR: "0x3B42d26E19FF860bC4dEbB920DD8caA53F93c600"
```

Comment:
@cygnusv , @KPrasch wdyt about using the deployer as the executor `lynx`? Open to other ideas.

Reply:
Yes, that's the most sensible choice for this, as well as Tapir